packages/storage/src/CredentialsStorageSchema.ts

```diff
@@ -20,13 +20,6 @@ export class CredentialsStorageSchema extends BaseStorageSchema<PersistedCredent
   protected override migrate(
     storageItem: IStorageItem<PersistedCredentials>,
   ): PersistedCredentials {
-    const storageVersion = storageItem.metadata.version;
-
-    if (this.version > storageVersion) {
-      return this.parseData({
-        refreshToken: '',
-      });
-    }
     return this.parseData(storageItem.data);
   }
```

Comment:
We can omit `migrate` I think.

Reply:
in the default migration method, it will throw if versions won't match, is that ok?

```
if (this.version !== storageVersion) {
      throw new NoMigrationPathError(this.key, storageVersion, this.version);
    }
```